android/BOINC/.idea/misc.xml

```diff
@@ -45,5 +45,5 @@
       </value>
     </option>
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_X" project-jdk-name="11" project-jdk-type="JavaSDK" />
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_16" project-jdk-name="11" project-jdk-type="JavaSDK" />
```

Comment:
This file is autogenerated and should be removed from the repo at all. Why did you make this change here?

Reply:
We have partial android studio project in our repo. In first I was object but now I understand the advantage of that. That it faster to load your android studio and it similar dev for all dev.